### DISCUSSION TITLE
Feature request: historical bundle simulation

### DISCUSSION THREAD

**Discussion by thegostep** [flashbots/pm#39] [searchers]
The relay currently only supports simulation going back approximately 64 blocks. It may be useful to use an archive node for simulation such that it supports arbitrary historical blocks.

**Comment by jparyani** [comment]
The relay is now running with partial archive nodes, with full archive data going back to around March 12th. This means you can simulate blocks >= 12030000

**Reply by thegostep** [reply]
Great, tracking this change here: https://github.com/flashbots/mev-relay-js/issues/28
